app/views/jobs/index.html.erb

```diff
@@ -1,5 +1,36 @@
 All Jobs for <strong><%= @company.name %></strong>
 
 <% @jobs.each do |job| %>
-  <ul><%= job.title %> at <%= job.company.name %></ul>
+<ul><%= link_to job.title, company_job_path(@company, job) %> at <%= job.company.name %>
+  <button type="button" class="btn btn-danger">
+    <%= link_to "Delete", company_job_path(@company, job), method: :delete %>
+  </button>
+</ul>
+<% end %>
+
+<% @company.contacts.each do |contact| %>
+<ul>
+  <%= contact.first_name %> <%= contact.last_name %></br>
+  <%= contact.position %></br>
+  <%= contact.email %></br>
+</ul>
+<% end %>
+
+<%= form_for @contact do |f| %>
```

Comment:
this form is a separate piece of functionality from the rest of this page, i suggest pulling this out to a partial